catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -6,6 +6,33 @@ We have a lot of work to do, and we’ve only just begun.
 
 ---
 
+# February 2019
+
+## What's new?
+
+From the feedback we've had over the past couple of months, we've been adding
+clarity and emphasis to the information on our library item pages.
+
+This will hopefully help with getting an overview of the item and assess it's
+relevance more effectively.
+
+What we've done:
+* The title is the first piece of information seen, along with a summary of
+  the object, including contributors and it's publication date.
+* The details about the item have been grouped in sections to help with the
+  scanning of the information.
+* The ordering of the information has been slightly tweaked on the feedback of
+  what people consider important.
+
+The search bar persists through the search journey, including when and
+item is viewed to allow for refinement of search, or to just start again,
+without having to go back to the results page.
+
+We are always collecting feedback, so if you have any
```

Comment:
@jennpb - do you think it would be too much to put the toggles in here?
Or maybe just a button / link / thingy that puts them into the search filters toggle?